backend/supabase/functions/_shared/utils/security-validator.ts

```
@@ -140,9 +140,17 @@ export class SecurityValidator {
 
   sanitizeInput(input: string): string {
     return input
-      .replace(/<[^>]*>/g, '') // Remove HTML tags
-      .replace(/[<>&"']/g, '') // Remove dangerous characters
+      // Remove null bytes and control characters (except newline, tab, carriage return)
+      .replace(/[\x00-\x08\x0B\x0C\x0E-\x1F\x7F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)